.github/workflows/ci.yml

```diff
@@ -100,21 +99,14 @@ jobs:
       - name: "Setup Python"
         uses: actions/setup-python@v4
         with:
```

Comment:
I think you also need to pass in the file with respect to which you want to identify the cache. Better to explicitly state it in any case (IMO) - https://github.com/actions/setup-python#caching-packages-dependencies

Reply:
Roger!